npc/other/Global_Functions.txt

```diff
@@ -475,3 +475,24 @@ function	script	F_GetTradeRestriction	{
 		.@trade$ += "NoAuction|";
 	}
 }
+
+//== Function F_MesItemInfo ===================================
+// Show the item name and a clickable link for the item description
+// Only works with mes and mesf, does not work in menu/select
+function	script	F_MesItemInfo	{
+	.@item = getarg(0);
+	.@itemname$ = getitemname(.@item);
+	if (.@itemname$ != "null") {
+		.@itemslot = getitemslots(.@item);
+		if (.@itemslot)
+			.@itemname$ = sprintf("%s [%d]", .@itemname$, .@itemslot);
```

Comment:
I knew you going to say this, I have tested this in my getitemname2 function
all items are default to 0 slots 
```
	Def: Defense                  (int, defaults to 0)
	Range: Attack Range           (int, defaults to 0)
	Slots: Slots                  (int, defaults to 0)
	Job: {                        (defaults to all job)
		All: true/false               (boolean, defaults to false)
		Novice: true/false            (boolean, defaults to false)
		Swordsman: true/false         (boolean, defaults to false)
		Magician: true/false          (boolean, defaults to false)
```

and no, items with [0] slot doesn't show in-game
please test it, 

and WHY the fuck I have to upload a screenshot every time ????
sigh...
OK I upload screenshots because you *think* it work your way but it doesn't work like that

```
prontera,155,185,5	script	asdasdad	1_F_MARIA,{
	getitem 1201, 1;
	getitem 1203, 1;
	mes(F_MesItemInfo(1201));
	mes(F_MesItemInfo(1203));
	close;
}
```
![fuck](https://raw.githubusercontent.com/AnnieRuru/customs/master/screenshots/screen2019Hercules004.jpg)